containers/Connector/Connector.tsx

```diff
@@ -56,7 +56,7 @@ const useConnector = () => {
 	);
 
 	useEffect(() => {
-		sdk.setProvider(provider).then(handleNetworkChange);
+		sdk.context.setProvider(provider).then(handleNetworkChange);
```

Comment:
What do you think about keeping the api as `sdk.setProvider`, and the sdk handling the routing there, as context feels more an internal thing to the sdk?

Reply:
True. Re-added the sdk setProvider methods.